generator/tree.go

```diff
@@ -145,6 +155,11 @@ func generateConfigModule(cfg *ModuleConfig, node *Node, nameToNode map[string]*
 			if !ok {
 				return // Unsupported type.
 			}
+
+			if !metricAccess(n.Access) {
+				return // Inaccessible metrics.
```

Comment:
After adding some debug log here , the whole test case output is:

```
WARN[0000] Error, root with  is inaccessible             source=tree.go:160
WARN[0000] Error, root with  is inaccessible             source=tree.go:160
WARN[0000] Error, root with  is inaccessible             source=tree.go:160
WARN[0000] Error, OCTETSTR with  is inaccessible         source=tree.go:160
WARN[0000] Error, INTEGER with  is inaccessible          source=tree.go:160
WARN[0000] Error, NETADDR with  is inaccessible          source=tree.go:160
WARN[0000] Error, IPADDR with  is inaccessible           source=tree.go:160
WARN[0000] Error, COUNTER with  is inaccessible          source=tree.go:160
WARN[0000] Error, GAUGE with  is inaccessible            source=tree.go:160
WARN[0000] Error, TIMETICKS with  is inaccessible        source=tree.go:160
WARN[0000] Error, COUNTER64 with  is inaccessible        source=tree.go:160
WARN[0000] Error, BITSTRING with  is inaccessible        source=tree.go:160
WARN[0000] Error, UINTEGER with  is inaccessible         source=tree.go:160
WARN[0000] Error, UNSIGNED32 with  is inaccessible       source=tree.go:160
WARN[0000] Error, INTEGER32 with  is inaccessible        source=tree.go:160
WARN[0000] Error, MacAddress with  is inaccessible       source=tree.go:160
WARN[0000] Error, tableNoAccess with ACCESS_NOACCESS is inaccessible  source=tree.go:160
WARN[0000] Error, tableWriteOnly with ACCESS_WRITEONLY is inaccessible  source=tree.go:160
WARN[0000] Error, tableNotify with ACCESS_NOTIFY is inaccessible  source=tree.go:160
WARN[0000] Error, tableIndex with  is inaccessible       source=tree.go:160
WARN[0000] Error, tableFoo with  is inaccessible         source=tree.go:160
WARN[0000] Error, octetIndex with  is inaccessible       source=tree.go:160
WARN[0000] Error, octetFoo with  is inaccessible         source=tree.go:160
WARN[0000] Error, bitstringIndex with  is inaccessible   source=tree.go:160
WARN[0000] Error, bitstringFoo with  is inaccessible     source=tree.go:160
WARN[0000] Error, ipaddrIndex with  is inaccessible      source=tree.go:160
WARN[0000] Error, ipaddrFoo with  is inaccessible        source=tree.go:160
WARN[0000] Error, netaddrIndex with  is inaccessible     source=tree.go:160
WARN[0000] Error, netaddrFoo with  is inaccessible       source=tree.go:160
WARN[0000] Error, physaddress48Index with  is inaccessible  source=tree.go:160
WARN[0000] Error, physaddress48Foo with  is inaccessible  source=tree.go:160
WARN[0000] Error, octetFoo with  is inaccessible         source=tree.go:160
```

I'm still wondering the result...